Grant Application - Reference blockchain system.md

```diff
@@ -0,0 +1,286 @@
+- **Costs:** 3,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | We will design a new reference license in our project, which records and reflects the reference relationship between the referred old works and the new works that are based on the old works. |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains the design and usage of our work in details. |
+|  1. | Specification | We will specify the design and management of our DAO community where users can participant in and contribute to the governance of our reference system equally and directly. |
+
+
+### Milestone 6 — System building and testing
+
+- **Estimated duration:** 1 month
+- **FTE:**  2 developer, 1 technical researcher, 1 economic researcher, and 1 management researcher
+- **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | We will design a new reference license in our project, which records and reflects the reference relationship between the referred old works and the new works that are based on the old works. |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains the design and usage of our work in details. |
+|  1. | Specification | We will combine all our previous works above together to build the integrated system. We will specify the configurations and implementations of the whole system, and encourage everyone within the community to use, test and improve. |
+|  2. | Setup | We provide a repository that forms the basis of our deliverables. |
+|  3. | Core Types | We provide the core data and coding types in details. |
+|  4. | User Testing | We will make the testing plan and invite users for testing and evaluation. After that we will produce testing report, and improve our system continually based on that. |
```

Comment:
Can you be more specific here? What will you be measuring? Who will be your cohort?

Reply:
We will mainly measure the degree and utility of decentralization in Web3 after implementing our project. How decentralized Web3 could be after using our system, and how online services could be better on such more decentralized Web3. We will also test the functionalities, security, and performance of the demo we built for our system. The cohort will be the Web3 users.